Restfull API/api/tests/test_auth.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import os
+import json
+import sys
+import inspect
+
+currentdir = os.path.dirname(os.path.abspath(
+  inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
```

Comment:
What is line 7 to line 10 for?

Reply:
I had problems importing some functions from my other modules, so I used that code to give my text file the intended path